### ISSUE TITLE
Some source code files missing in the output build

### ISSUE THREAD

**Issue by shipmints** [jimeh/emacs-builds#40]
e.g., nsterm.m isn't in Resources/src but the nsterm.h file does get propagated. Curious.

**Comment by jimeh** [comment]
Ah, yeah that's an oversight on my part. The sources embedding code only picks up `*.c` and `*.h` files from Emacs' `src` directory. Are there any other extensions aside from `*.m` that should be included added to the list?

**Comment by shipmints** [comment]
All the ChangeLogs, the .dbxinit, .gdbinit files, the xbm bitmaps. Why not just blindly copy everything? I think that's what should be done, actually. If your build (I haven't looked) "shites where it sleeps" and speckles object files all over the src three then either change the autoconf output dir to shite somewhere else or copy a pristine src tree separately?

**Comment by jimeh** [comment]
Fair point. Originally I included the source files just for the sake of `M-x describe-function` and similar for stuff defined in C to actually be able to link to and show the source. So I wasn't very concerned about files that wouldn't be needed for that.

**Comment by jimeh** [comment]
Yep, the C sources `src` folder included in Emacs.app now is grabbed after all patches are applied, but before the configure script has run.

The only thing I filter out and don't include are the `Changelog.1` though `Changelog.13` files, as they're 6.7 MB in size, and cover 1985 to 2015, and seems kinda meh to include in every single build.

**Comment by jimeh** [comment]
As for 29.4, in theory you could run my build script locally with default options, having it build the `emacs-29.4` tag, and then just ctrl+c once it's done applies patches, and you see it running autoconf or the configure script. Then you'll find the C sources under the relevant source tree it extracted.

**Comment by shipmints** [comment]
Apologies if we talked about this already, but do you want to produce a nix-based 29.4 build that includes all the sources? I'd switch to that one, myself, debugging nsterm.m nsfns.m etc is a tad of a pain.

**Comment by jimeh** [comment]
Ah yes, a while ago I had fired off a Nix-based build against 30.0.91 to test pushing newer artifacts into existing releases, and that did seem to work fine.

I've now fired off a new build [here](https://github.com/jimeh/emacs-builds/actions/runs/12189972218) of 29.4. When done, it should simply upload new artifacts targeting macOS 11 to the existing [29.4 release](https://github.com/jimeh/emacs-builds/releases/tag/Emacs-29.4).

I'll update the Homebrew Cask for 29.4 tomorrow as well after I've had a chance to do some testing.
